google-cloud-storage/src/test/java/com/google/cloud/storage/jqwik/StorageArbitraries.java

```diff
@@ -137,4 +151,48 @@ public String get() {
       return value;
     }
   }
+
+  public static Objects objects() {
+    return Objects.INSTANCE;
+  }
+
+  public static final class Objects {
+    private static final Objects INSTANCE = new Objects();
+
+    private Objects() {}
+
+    public Arbitrary<String> storageClass() {
+      // TODO: return each of the real values and edge cases (including invalid values)
+      return Arbitraries.strings().all().ofMinLength(1).ofLength(1024);
+    }
+
+    public Arbitrary<ObjectChecksums> objectChecksumsArbitrary() {
+      return Combinators.combine(
+              Arbitraries.integers().greaterOrEqual(1),
+              Arbitraries.strings()
+                  .map(
+                      s ->
+                          BaseEncoding.base64()
+                              .encode(Hashing.md5().hashBytes(s.getBytes()).asBytes())))
+          .as(
```

Comment:
This can be done in a later PR if you feel like it.

This will need to changes from a combine of two arbitraries, to a single arbitrary which yields both crc32c and md5 values.

We want to ensure the value for those fields is generated base on the same bytes in the general case. Later we can investigate if we want to add an edge case where the md5 does not correspond to crc32c of the same input data.

Reply:
ah I see makes sense. I will look into it.